documentation/docs/25-build-and-deploy/50-adapter-static.md

```diff
@@ -170,4 +170,4 @@ jobs:
         uses: actions/deploy-pages@v4
 ```
 
-If you're not using GitHub actions to deploy your site (for example, you're pushing the built site to its own repo), add an empty `.nojekyll` file in your `static` directory to prevent Jekyll from interfering.
+If you're not using GitHub actions to deploy your site (for example, you're pushing the built site to its own repo), add an empty `.nojekyll` file in your `static` directory to prevent Jekyll from interfering. If you use `gh-pages` to push the build folder to your repository, you need to add the `-t` option to the command `npx gh-pages -d build -t` to include dot files (e.g., `.nojekyll`) in your build branch.
```

Comment:
```suggestion
If you're not using GitHub actions to deploy your site (for example, you're pushing the built site to its own repo), add an empty `.nojekyll` file in your `static` directory to prevent Jekyll from interfering. Alternatively, if you're using the [`gh-pages`](https://www.npmjs.com/package/gh-pages) cli, you should add the `--nojekyll` flag.
```

Slight tweak to use this option instead, as may reduce accidentally committing a dotfile you didn't intend